typings/externs.d.ts

```diff
@@ -33,9 +33,17 @@ declare global {
       mixedContent: boolean;
       verbose: boolean;
       quiet: boolean;
+      disableDeviceEmulation?: boolean;
+      disableCpuThrottling?: boolean;
+      disableNetworkThrottling?: boolean;
     }
 
-    export interface Config {}
+    export interface Config {
```

Comment:
hm maybe we need to rename this interface or move these props somewhere else? those properties are on the pass rather than the root config?

Reply:
ah, good point. I have a branch where I've started to split this up in configjson vs config, and then we'll have pass config info, but this at least probably shouldn't be here for now

Reply:
moved them to `LH.ConfigPass`. Still hacky until we get real type checking for the config, but at least not both hacky and wrong :)